modules/component/spec/core/utils/is-zone-less.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { isZoneLess } from '../../../src/core/utils';
+
+class NoopNgZone {}
+
+describe('isZoneLess', () => {
+  it('should return false is something else than noop zone is passed', () => {
+    expect(isZoneLess({})).toBe(false);
+  });
+
+  it('should return false is something else than noop zone is passed', () => {
```

Comment:
This should probably say something like "should return true if a noop zone is passed".

Reply:
done